src/v2/Apps/Collect/Routes/Collection/Components/CollectionArtworksFilter.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import { RelayRefetchProp, createRefetchContainer, graphql } from "react-relay"
+import { BaseArtworkFilter } from "v2/Components/ArtworkFilter"
+import {
+  ArtworkFilterContextProvider,
+  Counts,
+  SharedArtworkFilterContextProps,
+} from "v2/Components/ArtworkFilter/ArtworkFilterContext"
+import { updateUrl } from "v2/Components/ArtworkFilter/Utils/urlBuilder"
+import { usePathnameComplete } from "v2/Utils/Hooks/usePathnameComplete"
+import { useRouter } from "v2/System/Router/useRouter"
+import { CollectionArtworksFilter_collection } from "v2/__generated__/CollectionArtworksFilter_collection.graphql"
+
+interface CollectionArtworksFilterProps {
+  relay: RelayRefetchProp
+  collection: CollectionArtworksFilter_collection
+  aggregations?: SharedArtworkFilterContextProps["aggregations"]
+  counts?: Counts
+}
+
+export const CollectionArtworksFilter: React.FC<CollectionArtworksFilterProps> = props => {
+  const { relay, collection, aggregations, counts } = props
+  const { slug } = collection
+
+  const { match } = useRouter()
+  const { pathname } = usePathnameComplete()
+
+  return (
+    <ArtworkFilterContextProvider
+      // Reset state of filter context without calling reset; which would
+      // affect analytics.
+      key={pathname}
+      filters={match.location.query}
+      sortOptions={[
+        { text: "Default", value: "-decayed_merch" },
+        {
+          text: "Price (desc.)",
+          value: "sold,-has_price,-prices",
```

Comment:
I'm not familiar with these sort options. Why do we have three values here?

Reply:
This value [was specified](https://github.com/artsy/force/pull/8005/files#diff-0e882eaf2d4c702041d1a6a2ec52c47a931b5f44cbebd4ec1637284832aaea2fL137-L138) for the sort filter. I just moved the existing code to a separate component to use `createRefetchContainer`